src/Tools.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Marem\PayumPaybox;
+
+/**
+ * Class Tools.
+ *
+ * @author Lexik <dev@lexik.fr>
+ * @author Olivier Maisonneuve <o.maisonneuve@lexik.fr>
+ */
+class Tools
+{
+    /**
+     * Makes an array of parameters become a querystring like string.
+     *
+     * @return string
+     */
+    public static function stringify(array $array)
```

Comment:
What is the difference with [http_build_query PHP function](https://www.php.net/manual/en/function.http-build-query.php) ?

Reply:
Right, I refactored this. This code comes from https://github.com/lexik/LexikPayboxBundle/blob/master/Paybox/System/Tools.php wich was code with php 5 and the function was already there.